src/Bicep.Core/Emit/EmitLimitationCalculator.cs

```diff
@@ -706,5 +707,33 @@ private static void BlockCaseInsensitiveNameClashes<T>(string itemTypePluralName
                     item => DiagnosticBuilder.ForPosition(nameSyntaxExtractor(item)).ItemsMustBeCaseInsensitivelyUnique(itemTypePluralName, clashingNames)));
             }
         }
+
+        private static void BlockSymbolicReferencesInInnerScopedNestedDeployments(SemanticModel model, IDiagnosticWriter diagnostics)
+        {
+            var innerScopedEvaluationIsDefault = model.EmitterSettings.EnableSymbolicNames;
+
+            foreach (var resource in model.DeclaredResources.Where(r => r.IsAzResource))
+            {
+                if (!LanguageConstants.ResourceTypeComparer.Equals(resource.TypeReference.FormatType(), AzResourceTypeProvider.ResourceTypeDeployments) ||
+                    resource.Symbol.DeclaringResource.GetBody().TryGetPropertyByName("properties", StringComparison.OrdinalIgnoreCase)?.Value is not ObjectSyntax propertiesObject ||
```

Comment:
>GetBody [](http://example.com/codeflow?start=54&length=7)

Does this need to be `TryGetBody()`? Will this logic ever run if the resource body is malformed or missing?